src/services/SiteDashboardService.ts

```diff
@@ -35,3 +36,10 @@ export const updateViewedReviewRequests = async (
   const endpoint = `/sites/${siteName}/review/viewed`
   return apiService.post(endpoint)
 }
+
+export const getSiteLaunchStatus = async (
+  siteName: string
+): Promise<SiteLaunchDto> => {
+  const endpoint = `/sites/${siteName}/launchInfo`
+  return apiService.get(endpoint)
```

Comment:
do we not need a `.then((res) => res.data)`?

Reply:
this tripped me in the later PRs :/ thanks for the catch !
1205cb98b7e60d37b8efce04262cd2e422100151